python/semantic_kernel/planners/function_calling_stepwise_planner/function_calling_stepwise_planner_options.py

```diff
@@ -13,15 +15,15 @@
 class FunctionCallingStepwisePlannerOptions(PlannerOptions):
     """The Function Calling Stepwise Planner Options."""
 
-    max_tokens: Optional[int] = None
-    max_tokens_ratio: Optional[float] = 0.1
-    max_completion_tokens: Optional[int] = None
-    max_prompt_tokens: Optional[int] = None
-    get_initial_plan: Optional[Callable[[], str]] = None
-    get_step_prompt: Optional[Callable[[], str]] = None
-    max_iterations: Optional[int] = 15
-    min_iteration_time_ms: Optional[int] = 100
-    execution_settings: Optional[OpenAIPromptExecutionSettings] = None
+    max_tokens: int | None = None
```

Comment:
Is `Optional` no longer the way to do this?

Reply:
Now that we're starting to use `from __future__ import annotations` in the codebase, we get the nicety of being able to use the newer type hints. In Python 3.8/3.9, type hints like `int | None` are treated as string literals, and their evaluations are deferred. Python 3.8/3.9 will see the `|` operator as a union type, similar to Optional[int].

Reply:
Ok